util/tlgen/lib.py

```diff
@@ -150,7 +156,7 @@ def find_pow2_size(addr: Dict[Any, Any],
     should be less than or equal to 0x1_0000. Cannot be 0x4_0000. So, this
     case, the function returns the original min_size value 0x21000.
     """
-    base_addr = int(addr["base_addr"], 0)
+    base_addr = int(addr, 0)
```

Comment:
This is weird: if addr is a dict then int(addr, 0) should give an error since int would expect addr to be a string. The caller seems to pass a string, so what is happening with the type annotation?

Reply:
Good point. I think the type annotation was added in the meanwhile on master. I updated the annotation to let `addr` be a string. The function is only used once.